src/main/scala/za/co/absa/abris/avro/registry/AbstractAbrisRegistryClient.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.abris.avro.registry
+
+import io.confluent.kafka.schemaregistry.avro.AvroSchema
+import io.confluent.kafka.schemaregistry.client.{SchemaMetadata, SchemaRegistryClient}
+import org.apache.avro.Schema
+
+import java.util
+
+
+abstract class AbstractAbrisRegistryClient(client: SchemaRegistryClient) extends AbrisRegistryClient {
```

Comment:
Could this be renamed to `AbstractConfluentRegistryClient`?

*Abris Client* -  not dependent on implementation
*Confluent Client* - depends on confluent

Reply:
Yeah that makes sense